data/project-templates/js-node/src/index.js

```diff
@@ -0,0 +1,39 @@
+import * as dotenv from "dotenv";
+import { LogLevel, ProposalFilters, TaskExecutor } from "@golem-sdk/golem-js";
+
+dotenv.config();
```

Comment:
I wonder if this is necessary in this snippet? Or maybe you need to add a comment that you can put YAGNA_APPKEY in the .env or specify it in yagnaOptions...

Reply:
This is copied from the example supplied by Grzesiek